Pod/Classes/PINRemoteImageManager.m

```diff
@@ -1421,9 +1430,7 @@ - (void)pin_addOperationWithQueuePriority:(PINRemoteImageManagerPriority)priorit
 {
     NSBlockOperation *operation = [NSBlockOperation blockOperationWithBlock:block];
     operation.queuePriority = operationPriorityWithImageManagerPriority(priority);
-    if ([operation respondsToSelector:@selector(qualityOfService)]) {
-        operation.qualityOfService = NSOperationQualityOfServiceBackground;
-    } else {
+    if (supportsQOS == NO) {
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wdeprecated-declarations"
         operation.threadPriority = 0.2;
```

Comment:
It's absolutely a guess. If you have ideas for imperially figuring out the best value, I'd love to hear them!